string command
string type
string path
---
int16 ack
